### ISSUE TITLE
Improve App Performance in pagespeed.web.dev

### ISSUE THREAD

**Issue by harveyjavier** [sparkpointio/sparkagent-launchpad#64]
I checked the app by inputting https://test.sparkpoint.io on [pagespeed.web.dev](https://pagespeed.web.dev/)

So far the app complies under the _Best Practices_ and _SEO_ both on Mobile and Desktop, but fails under _Performance_ and _Accessibility_
- [Mobile](https://pagespeed.web.dev/analysis/http-test-sparkpoint-io/hklnqp03wb?form_factor=mobile) 
  ![Image](https://github.com/user-attachments/assets/63d2950e-13ea-4de4-a51d-1c77c809a0e9)
  ![Image](https://github.com/user-attachments/assets/7b229134-b00c-4155-946b-148edcfcc692)
- [Desktop](https://pagespeed.web.dev/analysis/http-test-sparkpoint-io/hklnqp03wb?form_factor=desktop)
  ![Image](https://github.com/user-attachments/assets/ec2303bb-8175-40d5-8f65-25c9001017b5)
  ![Image](https://github.com/user-attachments/assets/07ca877c-fe0a-478a-9c68-234a212e8d0b)

Please improve the _Performance_ and _Accessibility_, and make sure we achieve 90 above in both sections.

**Comment by harveyjavier** [comment]
We just need to hit the _green_ level on the _Performance_ section to close this issue

**Comment by harveyjavier** [comment]
#103 is enough fix for now. Will be closing this one @sparkpointio/core
